pkg/image/apiserver/admission/apis/imagepolicy/v1/default-policy.yaml

```diff
@@ -12,6 +12,7 @@ executionRules:
   onResources:
   - resource: pods
   - resource: builds
+    group: build.openshift.io
```

Comment:
@bparees this was the culprint... i think we should always require the group and not try to guess here

Reply:
this had nothing to do with it.  I experienced the behavior testing on a real cluster, not running the test.

Reply:
well I see this during the test run as well ;-)

Reply:
@bparees can you retest with this change?

Reply:
(ie this may be necessary now w/ your fix, but it wasn't the cause of the bug)